<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diaryDetail</class>
 <widget class="QWidget" name="diaryDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="destinationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>destination</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="contentBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitPointButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>提交评分</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pointSpin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>觉得不错，想给日记评分？</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>point</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="browseLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
